lib/Onnxifi/Base.cpp

```diff
@@ -48,22 +48,64 @@ onnxStatus Graph::initGraph(const void *onnxModel, size_t onnxModelSize,
   // TODO: support multiple functions here.
   function_ = backendPtr_->getEE().getModule().createFunction("inference");
 
-  std::unique_ptr<ModelLoader> loader = ModelLoader::parse(
-      onnxModel, onnxModelSize, weightCount, weightDescriptors, *function_);
+  // Keep loader to have an access to the input and output ONNX vars.
+  // The loader itself is not necessary we can just keep mapping
+  // ONNX var name => Glow actual var.
+  loader_ = ModelLoader::parse(onnxModel, onnxModelSize, weightCount,
+                               weightDescriptors, *function_);
   // TODO: make better error reporting.
-  if (!loader) {
+  if (!loader_) {
     return ONNXIFI_STATUS_INTERNAL_ERROR;
   }
 
   return ONNXIFI_STATUS_SUCCESS;
 }
 
-onnxStatus Graph::run() { return ONNXIFI_STATUS_SUCCESS; }
+onnxStatus Graph::run() {
+  // TODO: copy all inputs from the addresses specified in the inputs_
+  // to Glow tensors and execute the inference.
+
+  // TODO: run inference.
+
+  // TODO: copy outputs to the addresses specified in the outputs_.
+  return ONNXIFI_STATUS_SUCCESS;
+}
+
+static onnxStatus
+processDescriptors(uint32_t count, const onnxTensorDescriptorV1 *descriptors,
+                   std::unordered_map<glow::Variable *, onnxPointer> &mapping,
+                   const ModelLoader &loader) {
+  for (unsigned i = 0; i < count; ++i) {
```

Comment:
It's OK for now. As a TODO, we should check the dimensions of input/output tensors and track whether it has changed.